<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Station</class>
 <widget class="QDialog" name="Station">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>328</width>
       <height>266</height>
      </size>
     </property>
     <property name="title">
      <string>State Panel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="lst_State"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_00">
        <property name="text">
         <string>CHOOSE DESTINATION STATION</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="destStation">
        <item>
         <property name="text">
          <string>Station 1 (COM3, COM4)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Station 2 (COM5, COM6)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Station 3 (COM7, COM8)</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Input/Output Panel</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lblSource">
          <property name="text">
           <string>Source Station</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ln_Input"/>
        </item>
        <item>
         <widget class="QLabel" name="lblDest">
          <property name="text">
           <string>Destination Station</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lst_Output"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
